qsiprep/workflows/fieldmap/base.py

```diff
@@ -264,16 +260,17 @@ def init_sdc_wf(fieldmap_info, dwi_meta):
 
     # FIELDMAP-less path
     if fieldmap_info['suffix'] == 'syn':
+        # XXX: Should be replaced with SynB0.
         syn_sdc_wf = init_syn_sdc_wf(
             bold_pe=dwi_meta.get('PhaseEncodingDirection', None),
         )
 
         workflow.connect([
             (inputnode, syn_sdc_wf, [
                 ('t1_brain', 'inputnode.t1_brain'),
+                # Only place a template transform is used on DWI data
                 ('t1_2_mni_reverse_transform', 'inputnode.t1_2_mni_reverse_transform'),
                 ('b0_ref', 'inputnode.bold_ref'),
-                ('template', 'inputnode.template'),
```

Comment:
I just want to note this. When we switch to `--output-spaces`, we should (1) raise an error if users try to use syn with infant data and (2) only pass along the 2009c template transform here.